imports/models/BoardFavoritesCollection.ts

```diff
@@ -0,0 +1,30 @@
+import { BaseCollection } from './BaseCollection'
+import { BoardRecord } from './BoardCollection'
+
+export interface BoardFavoriteRecord {
+  userid: string
+  boardId: string
+}
+
+export class BoardFavoritesCollection extends BaseCollection<BoardFavoriteRecord> {
+  constructor() {
+    super('board_favorites')
+  }
+
+  public async addFavoriteBoard(userid: string, boardId: string): Promise<void> {
+    await this.insert({ userid, boardId })
```

Comment:
use upsert here is better to avoid insert duplicate records.
```
this.collection.update({ userid, boardId}, { userid, boardId }, { upsert: true })
```